views/index.ejs

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Gazette Notice</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href='/style.css'>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+</head>
+<body>
+  <main class="main-container">
+    <header class="header">
+      <button class="header__btn header__btn--page"><span>1</span>-<span>10</span></button>
+      <h1 class="header__title">Notices</h1> 
+      <button class="header__btn header__btn--next" role="navigation">next>></button>

```

Comment:
The icons in `next >>` could be added via CSS `:after` pseudo element, which would keep the text free from clutter for screen readers